src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/ActiveDirectoryTests.ps1

```diff
@@ -588,12 +588,78 @@ function Test-NewADServicePrincipalWithoutApp
     Assert-Throws { Remove-AzureRmADServicePrincipal -ObjectId $servicePrincipal.Id -Force}
 }
 
+<#
+.SYNOPSIS
+Tests creating a service principal with reader permissions
+#>
+function Test-NewADServicePrincipalWithReaderRole
+{
+	# Setup
```

Comment:
can we add another test where in we dont specify any role or scope and make sure that no roleassignment change happens before and after running the command

Reply:
@darshanhs90 add a check in `Test-NewADServicePrincipalWithoutApp` that no role assignment was created for the new service principal